### PR TITLE
Fix UtilTest#createDirectoriesInRoot() on mac

### PR DESCRIPTION
**Pull request by timja**
```
[ERROR] Tests run: 27, Failures: 0, Errors: 1, Skipped: 2, Time elapsed: 0.247 s <<< FAILURE! - in hudson.UtilTest
[ERROR] createDirectoriesInRoot  Time elapsed: 0.012 s  <<< ERROR!
java.nio.file.FileSystemException: /new-dir-in-root: Read-only file system
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:100)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:389)
	at java.base/java.nio.file.Files.createDirectory(Files.java:690)
	at hudson.Util.createDirectories(Util.java:1785)
	at hudson.UtilTest.createDirectoriesInRoot(UtilTest.java:676)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
```

### CODE REVIEW

File: core/src/test/java/hudson/UtilTest.java
Comment (timja) [inline core/src/test/java/hudson/UtilTest.java:679]: Could also adjust the `assumeFalse` a few lines above if we don't want to change this.

This is a superclass of `AccessDeniedException`

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
